src/doc/src/reference/specifying-dependencies.md

```diff
@@ -461,6 +461,19 @@ Like with Rust, the syntax here supports the `not`, `any`, and `all` operators
 to combine various cfg name/value pairs. Note that the `cfg` syntax has only
 been available since Cargo 0.9.0 (Rust 1.8.0).
 
+If you want to know which cfg targets are available on your platform, run
+`rustc --print=cfg` from the command line. If you want to know which `cfg`
+targets are available for another platform, such as 64-bit Windows,
+run `rustc --print=cfg --target=x86_64-pc-windows-msvc`.
+If you want to know which cfg targets are used by the release profile,
```

Comment:
I would remove the note about `-O`.  Cargo doesn't use the profile when detecting `cfg` settings.  Also, AFAIK, it would only affect the `debug_assertions` setting.

Reply:
You're right.

```
PS> cat Cargo.toml
[package]
name = "test-project"
version = "0.1.0"
authors = ["Michael Howell <michael@notriddle.com>"]
edition = "2018"

[dependencies]

[target.'cfg(debug_assertions)'.dependencies]
regex = "1"
PS> cargo build --release
   Compiling memchr v2.2.0
   Compiling regex v1.1.6
   Compiling ucd-util v0.1.3
   Compiling lazy_static v1.3.0
   Compiling utf8-ranges v1.0.2
   Compiling thread_local v0.3.6
   Compiling regex-syntax v0.6.6
   Compiling aho-corasick v0.7.3
   Compiling test-project v0.1.0 (C:\Users\micha\IdeaProjects\test-project)
    Finished release [optimized] target(s) in 38.74s
PS>
```

It looks like it's https://github.com/rust-lang/cargo/issues/5777 that's responsible for this.